report/renderer/details-renderer.js

```diff
@@ -472,15 +471,16 @@ export class DetailsRenderer {
   }
 
   /**
-   * @param {LH.Audit.Details.List} details
+   * @param {LH.FormattedIcu<LH.Audit.Details.List>} details
    * @return {Element}
    */
   _renderList(details) {
     const listContainer = this._dom.createElement('div', 'lh-list');
 
     details.items.forEach(item => {
-      const snippetEl = SnippetRenderer.render(this._dom, item, this);
-      listContainer.appendChild(snippetEl);
+      const listItem = this.render(item);
+      if (!listItem) return;
+      listContainer.appendChild(listItem);
```

Comment:
was gonna suggest `append` over `appendChild` but I see we use it in so many places in report anyway.  #13991

Reply:
> was gonna suggest `append` over `appendChild` but I see we use it in so many places in report anyway.  #13991

it's always a good time to prefer `append` :)